forms.py

```diff
@@ -0,0 +1,32 @@
+from flask_wtf import FlaskForm
+from models import ConciergeMii
+from wtforms import StringField, SubmitField, PasswordField
+from wtforms.validators import ValidationError, DataRequired
+class LoginForm(FlaskForm):
+    username = StringField('Username')
+    password = PasswordField('Password')
+    submit = SubmitField("Enter the underground")
+class KillMii(FlaskForm):
+    # Form for deleting a concierge mii
+    miiid = StringField('Mii ID',validators=[DataRequired()])
+    submit = SubmitField('Delete!')
+class ConciergeForm(FlaskForm):
+    miiid = StringField('Mii ID',validators=[DataRequired()])
+    title = StringField('Concierge Mii Title', validators=[DataRequired()])
+    color1 = StringField('Shirt Color (RRGGBB)', validators=[DataRequired()])
+    color2 = StringField('Pants Color (RRGGBB)', validators=[DataRequired()])
+    message1 = StringField('Message 1', validators=[DataRequired()])
+    message2 = StringField('Message 2', validators=[DataRequired()])
+    message3 = StringField('Message 3', validators=[DataRequired()])
+    message4 = StringField('Message 4', validators=[DataRequired()])
+    message5 = StringField('Message 5', validators=[DataRequired()])
+    message6 = StringField('Message 6', validators=[DataRequired()])
+    message7 = StringField('Message 7', validators=[DataRequired()])
```

Comment:
These are kind of odd - I haven't really figured out how to enter them and would love to work with you on figuring out a better method. There are separate message "types" that play at different types, with their own seqs.

For example:
```xml
<msginfo>
  <type>1</type>
  <msglist>
    <seq>1</seq>
    <msg>Hello, with a normal face.</msg>
    <face>1</face>
    <seq>2</seq>
    <msg>I will now pretend to be scared.</msg>
    <face>5</face>
  </msglist>
</msginfo>
```

While the database schema currently does account for this, it's quite hard to represent. I was thinking of creating some custom drag-and-drop list with messages but am unsure on how easy that would be to enter.